### ISSUE TITLE
'Finder" doesn't work anymore.

### ISSUE THREAD

**Issue by Delice0** [nvimdev/lspsaga.nvim#1315]
### Describe the bug

Nothing happens when using 'Finder'.

### Steps to reproduce

active 'Finder' on some code.

### Expected behavior

expect to see a window popup with references/implementations etc.

### Neovim version (nvim -v)

latest

### lspsaga commit

latest

### Terminal name/version

kitty latest

**Comment by glepnir** [comment]
I can't reproduce. provide more info.

**Comment by glepnir** [comment]
could you provide a code snippet ? which language

**Comment by Delice0** [comment]
After latest changes, I don't seem to experience any issues.

P.s. can you please stop closing issues when they are not fixed? Or no conclusions are made?

**Comment by glepnir** [comment]
because i need reproduce step.

**Comment by avegancafe** [comment]
I'm also seeing this, @Delice0 did you ever figure this out? @glepnir what debug steps should I perform to help debug this?  E.g. when I open `:Lspsaga finder`, I see the loading dots, and then get this empty floating window:
<img width="1186" alt="image" src="https://github.com/nvimdev/lspsaga.nvim/assets/2652762/7f0b7349-ffe3-4f06-91c8-060c0ec199b2">

**Comment by Delice0** [comment]
I haven't found a solution for this issue therefor I've deleted the plugin and went with Aerial mainly because of the neat outline functionality.

**Comment by avegancafe** [comment]
Whoa good recommendation @Delice0 ! The outline and floating term, along with this finder window were the only parts of lspsaga that I was using. With the outline replaced, now I can finally replac the terminal with [fterm.nvim](https://github.com/numToStr/FTerm.nvim), the outline with [aerial.nvim](https://github.com/stevearc/aerial.nvim), and the finder with telescope lsp references!

**Comment by asao-vs** [comment]
I also reproduced the bug where :Lspsaga finder does not display information in the following environment:

> NVIM v0.10.0-dev-772+g0211f889b-Homebrew
> Build type: Release
> LuaJIT 2.1.0-beta3

Upon checking the operation with each commit, it was found that this commit (https://github.com/nvimdev/lspsaga.nvim/commit/dc5d7f7390569219a88a21cb46eff399e0ac0d90) is the cause. For now, I resolved the issue by specifying the commit right before it (db74412e0282505ea1e63a42060728fb74c1968e).

I don't have time to fix it right now, so I'm just sharing the information for now.
